Library/Formula/rem.rb

```diff
@@ -0,0 +1,24 @@
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  # version "20150618"
+  sha256 "e57173a26d2071692d72f3374e36444ad0b294c1284e3b28706ff3dbe38ce8af"
+
+  depends_on :macos => :mountain_lion
+  depends_on :xcode
```

Comment:
Is this runtime as well as for compile? i.e. if I install this, and then remove Xcode, will it work or not?